spring-boot/src/test/java/org/springframework/boot/logging/LoggingApplicationListenerTests.java

```diff
@@ -279,6 +280,23 @@ public void parseLevelsCaseInsensitive() throws Exception {
 	}
 
 	@Test
+	public void parseLevelsCaseInsensitivePassesTurkeyTest() throws Exception {
+		Locale original = Locale.getDefault();
```

Comment:
Thanks, just checking, no worries.
Yes, the existing solution works as well, but you have to remember to reset it when you're done.
Anyway, it's fine, just wanted to mention it. :+1: